src/client/app/components/groups/GroupsDetailComponent.tsx

```diff
@@ -3,77 +3,29 @@
+import TooltipMarkerComponent from '../TooltipMarkerComponent';
 import CreateGroupModalComponent from './CreateGroupModalComponent';
-import { GroupDefinition } from 'types/redux/groups';
-import * as _ from 'lodash';
-import HeaderComponent from '../../components/HeaderComponent';
-import { Dispatch } from 'types/redux/actions';
+import GroupViewComponent from './GroupViewComponent';
+import { authApi, authPollInterval } from '../../redux/api/authApi';
 
 /**
  * Defines the groups page card view
  * @returns Groups page element
  */
 export default function GroupsDetailComponent() {
-	const dispatch: Dispatch = useDispatch();
-
-	// Groups state
-	const groupsState = useSelector((state: State) => state.groups.byGroupID);
-	// Groups state loaded status
-	const groupsStateLoaded = useSelector((state: State) => state.groups.hasBeenFetchedOnce);
-	// The immediate children of groups is loaded separately.
-	const groupsAllChildrenLoaded = useSelector((state: State) => state.groups.hasChildrenBeenFetchedOnce);
 	// Check for admin status
-	const currentUser = useSelector((state: State) => state.currentUser.profile);
-	const loggedInAsAdmin = (currentUser !== null) && isRoleAdmin(currentUser.role);
-
-	// We only want displayable groups if non-admins because they still have
-	// non-displayable in state.
-	let visibleGroups;
-	if (loggedInAsAdmin) {
-		visibleGroups = groupsState;
-	} else {
-		visibleGroups = _.filter(groupsState, (group: GroupDefinition) => {
-			return group.displayable === true
-		});
-	}
-
-	// Units state
-	const units = useSelector((state: State) => state.units.units);
-	// Units state loaded status
-	const unitsStateLoaded = useSelector((state: State) => state.units.hasBeenFetchedOnce);
-	// Units state loaded status
-	const metersStateLoaded = useSelector((state: State) => state.meters.hasBeenFetchedOnce);
+	const isAdmin = useAppSelector(state => selectIsAdmin(state));
 
-	useEffect(() => {
-		// Note each modal is created for each group when the details are created so get all state now for all groups.
-		// Get meter details if needed
-		dispatch(fetchMetersDetailsIfNeeded())
-		// Makes async call to groups API for groups details if one has not already been made somewhere else, stores group ids in state
-		dispatch(fetchGroupsDetailsIfNeeded());
-		// TODO Is there a good way to integrate this into the actions so it must work correctly?
-		// You need the basic group state loaded since going to modify.
-		if (groupsStateLoaded) {
-			// Get all groups' meter and group immediate children into state. Since all modals done for all groups
-			// we get them all here.
-			dispatch(fetchAllGroupChildrenIfNeeded());
-		}
-		// In case the group state was not yet loaded you need to do this again.
-	}, [groupsStateLoaded, groupsAllChildrenLoaded]);
+	// page may contain admin info so verify admin status while admin is authenticated.
+	authApi.useTokenPollQuery(undefined, { skip: !isAdmin, pollingInterval: authPollInterval })
```

Comment:
I'm wondering why tis is not in admin, conversion & unit (and maps at some point) pages? Maybe others.

Reply:
Admin outlet has this polling effect so any of it's children also inherit the behavior.

Meter and group pages are kinda different because they modify behaviors based on auth status so they live outside the adminOutlet and explicitly need the polling added here